VTEX - Budgets API.json

```diff
@@ -0,0 +1,5235 @@
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Invalid request parameters."
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Authentication required."
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "Resource not found."
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/responseError"
+                },
+                "example": {
+                  "error": "An internal server error occurred."
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "VtexIdclientAutCookie": []
+          }
+        ]
+      }
+    },
+    "/api/budgets/{contextType}/{contextId}/{budgetId}/allocations/{allocationId}/reservations": {
+      "post": {
+        "tags": [
+          "Reservations"
+        ],
+        "summary": "Create reservation",
+        "description": "This endpoint creates a temporary reservation under a specific allocation in a VTEX budget.\n\n## Permissions\n\nThis endpoint does not require [License Manager resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/Content-Type"
+          },
+          {
+            "$ref": "#/components/parameters/contextType"
+          },
+          {
+            "$ref": "#/components/parameters/contextId"
+          },
+          {
+            "$ref": "#/components/parameters/budgetId"
+          },
+          {
+            "$ref": "#/components/parameters/allocationId"
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "description": "Request body for creating a new reservation.",
```

Comment:
It's not necessary a description on the root of the request or response body. You can remove them from all endpoints.